tree_segment.py

```diff
@@ -0,0 +1,434 @@
+import laspy
+import matplotlib.pyplot as plt
+import math
+import numpy as np
+import random
+import time
+import pandas as pd
+import argparse
+import re
+import os
+
+
+class Point_Cloud:
+    # object that stores the points in a point cloud, initialized from a las file
```

Comment:
Nit: Use docstrings rather than comments, and include a one-line summary followed by additional details: https://google.github.io/styleguide/pyguide.html#381-docstrings

Meta Commentary:
* Docstrings are python specific, but essentially just comments with additional features.
* "Nit" is usually unessential but nice to have. Ignoring, or replying "Ack." or making a quick change are all reasonable responses.
* In a project with an established style-guide, this might not be optional.

Reply:
Ack, done